### ISSUE TITLE
Whitelist domain [freshwallpapers.info] & [geckoandfly.com]

### ISSUE THREAD

**Issue by Nzzyme** [mitchellkrogza/Badd-Boyz-Hosts#12]
Hi,

I am using the "Unified hosts" from Steven Black which also includes one of your hosts "Mitchell Krog's - Badd Boyz Hosts".

A quick look and test seems there are many good sites that are on your block list. Not sure why they were blocked. Did you check if these sites are actually harmful?

Can you take a look at all your filters and remove the good ones from them? For now, please unblock the two in the title. I will continue to monitor and if more are found, I will have no other option but to completely remove this host itself. Thanks!

**Comment by mitchellkrogza** [comment]
Removed the two in your subject line in https://github.com/mitchellkrogza/Badd-Boyz-Hosts/commit/462f3a5b236bdf3bc23fac0bd4cf983b72d81a9d

Can you let me know which others you find which should not be here?

This list is originally compiled from my Nginx bad referrer blocker which are taken from logs of a number of sites I operate but it started out a year ago with an initial list from another repo which I am trying to iron out as I go along. 

I am now trying to separate the two projects from each other which I have already done but need to at least know which on here are considered OK and I can remove them no problem as the list that is used to generate this hosts files is now separated from the Bad Bot Blocker project.

A site like the freshwallpapers.info made it onto here as they link to content from my photography site without permission but as far as them being a "bad" host I agree they are not and have hence removed them in this latest commit. 

I will gladly remove any others if you spot any.

**Comment by mitchellkrogza** [comment]
If you want to help at all send a pull request on this file https://github.com/mitchellkrogza/Badd-Boyz-Hosts/blob/master/PULL_REQUESTS/domains.txt it is my intention to have an accurate list and not be blocking incorrect domains.

**Comment by Nzzyme** [comment]
That's not the point. You cannot expect us to clean up the mess that you have created. You host file needs to be immediately reviewed and taken off from Unified host. Not sure why was it included in the first place.

YOU need to review the list, check them if the links are valid/outdated/genuine and take an action.

**Comment by mitchellkrogza** [comment]
Then tell @StevenBlack to remove it, makes no difference in my life. If you
can't even provide domain names that you consider invalid then too bad. A
number of the other lists being used in unified have 50% domains that don't
even exist anymore. My list is from sites that referrer spam frequently as
I check logs daily which is where they come from. So whatever.

**Comment by mitchellkrogza** [comment]
@clemetine123 Pull Request sent for complete removal, are you happy now ?
@StevenBlack

**Comment by mitchellkrogza** [comment]
@clementine123 here's @StevenBlack response to removing my list from his unified hosts.

**Badd-Boyz is a good list with exceptional curation activity that comes only from dedicated, conscientious oversight that I wish all lists would possess.

So, no.

Closing.**
